[PATCH] optee-test: fix build failure with GCC 6

Add -Wno-unused-const-variable to disable errors introduced with the upgrade
to GCC 6:
error: 'pbkdf2_4_dkm' defined but not used [-Werror=unused-const-variable=]

Signed-off-by: Andrey Konovalov <andrey.konovalov@linaro.org>
diff --git a/host/xtest/Makefile b/host/xtest/Makefile
index 368d826..7960e5e 100644
--- a/host/xtest/Makefile
+++ b/host/xtest/Makefile
@@ -128,7 +128,8 @@ CFLAGS += -Wall -Wcast-align -Werror \
 	  -Wmissing-prototypes -Wnested-externs -Wpointer-arith \
 	  -Wshadow -Wstrict-prototypes -Wswitch-default \
 	  -Wwrite-strings \
-	  -Wno-missing-field-initializers -Wno-format-zero-length
+	  -Wno-missing-field-initializers -Wno-format-zero-length \
+	  -Wno-unused-const-variable
 endif
 
 CFLAGS += -g3
